ksql-engine/src/test/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactoryTest.java

```diff
@@ -118,13 +126,40 @@ public void shouldPickUpPrefixedSslConfig() {
 
     // When:
     final SchemaRegistryClient client =
-        new KsqlSchemaRegistryClientFactory(config, restServiceSupplier, sslFactory).create();
+        new KsqlSchemaRegistryClientFactory(config, restServiceSupplier, sslFactory,
+                                            Collections.EMPTY_MAP).create();
+
 
     // Then:
     assertThat(client, is(notNullValue()));
     EasyMock.verify(restService);
   }
 
+
+  @Test
+  public void shouldPassBasicAuthCredentialsToSchemaRegistryClient() {
+    // Given
+    final Map<String, Object> schemaRegistryClientConfigs = ImmutableMap.of(
+        "ksql.schema.registry.basic.auth.credentials.source", "USER_INFO",
+        "ksql.schema.registry.basic.auth.user.info", "username:password"
+    );
+
+    final KsqlConfig config = new KsqlConfig(schemaRegistryClientConfigs);
+
+    final Map<String, Object> expectedConfigs = schemaRegistryClientConfigs.entrySet()
+        .stream()
+        .map((e) -> new AbstractMap.SimpleEntry<>(
+              e.getKey().replaceFirst(KsqlConfig.KSQL_SCHEMA_REGISTRY_PREFIX, ""), e.getValue())
+        )
+        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
you could simplify this by applying the key transformation in the Collectors.toMap function